blog/_posts/btcpay-server-1-5-0.md

```diff
@@ -0,0 +1,117 @@
+
+![](/images/1-5-0-dashboard-1.png)
+
+![](/images/1-5-0-dashboard-2.png)
+
+Dashboard visually showcases critical store's data, providing you with a way to better understand your store's performance, so that you can make more informed decisions. Furthermore, a dashboard should help with easier management of recent invoices and pending payouts and refunds.
+
+Currently, the dashboard shows data for an on-chain payment method, pending payouts and refunds, recent invoices & transactions, and the data for the crowdfunding app.
+
+![](/images/1-5-0-dashboard-3.png)
+
+![](/images/1-5-0-dashboard-4.png)
+
+For more information on dashboard functionality, [check our docs](https://docs.btcpayserver.org/Dashboard/).
+
+We're building the dashboard with flexibility in mind, and the goal is extend its functionality via the plugin system. 
+
+We'd love to hear what do you think about the dashboard and how it works for you. Let us know what kind of data would you help you better manage your store.
+
+## Payout processor 🤖
+
+Payout processors are a new concept that hooks into the [pull payment system](https://docs.btcpayserver.org/PullPayments/), specifically the [payout part of it](https://docs.btcpayserver.org/PullPayments/#approve-and-pay-a-payout). This new functionality allows you to automate the flow of the payouts.
+
+![](/images/1.5.0-payout-processor-1.png)
+
+Currently, we support payout processors for on-chain and off-chain payment methods. You can schedule payouts for on-chain, bolt11, lightning address, lnurl, etc.
+
+Set up a time interval in your store settings upon which your payouts will be batched and broadcasted to the network. 
+
+![](/images/1.5.0-payout-processor-2.png)
+
+Payout processor, in its current state, allows you to save on transaction fees by batching the store's payments (wallet payments, refunds, payouts) and broadcasting them at a specific, pre-set time interval, for example, hourly, daily, weekly and more.
+
+### Schedule for later 📆
+
+We understand that 'payout processors' may sound a bit abstract, and that's why we've added a concrete use-case for it through our brand-new **schedule for later** function in the wallet.
+
+![](/images/1-5-0-schedule-transaction1.png)
+
+This means that now, instead of signing and broadcasting a transaction, you have the option to schedule it for later on at a pre-set time interval.
+
+And since our refunding system goes through the wallet, the scheduling option now allows you to batch your customer's refunds and save on fees.
+
+![](/images/1-5-0-schedule-transaction2.png)
+
+We plan to introduce a better permission system and hook it into the store so that in a particular store role, the store manager can create payouts, for example, refunds, but only the store owner can send them out.
```

Comment:
```suggestion
And since our refunding system goes through the pull payments system, if payout processors are configured on your store, they can now automatically be paid out.
```

Reply:
This is already this way, I suggest we don't mention the store roles proposal just yet.